src/main/java/cn/nukkit/math/MathHelper.java

```diff
@@ -92,7 +89,7 @@ public static int ceil(float floatNumber) {
     }
 
     public static int clamp(int check, int min, int max) {
-        return check > max ? max : (check < min ? min : check);
+        return check > max ? max : (Math.max(check, min));
```

Comment:
Not sure whether it's necessary here, since it's quite low level method

Reply:
Not necessary but it makes the code cleaner. Should I really remove this?